### PR TITLE
move to terraform 0.12

### PR DESCRIPTION
**Pull request by venturel**

### CODE REVIEW

**Review by minniux — APPROVED** [review]

File: user-data/main.tf [hunk 1]
```diff
@@ -78,15 +83,17 @@ runcmd:
...
-  environment  = "${var.environment}"
-  project      = "${var.project}"
-  service_type = "upstart"
+source = "github.com/skyscrapers/terraform-teleport//teleport-bootstrap-script?ref=3.3.6"
+auth_server = var.teleport_server
+auth_token = var.teleport_auth_token
+function = "ecs"
+environment = var.environment
+project = var.project
+service_type = "upstart"
```
Comment (duboisph) [inline user-data/main.tf:97]: Not part of this PR, but in a next release we should move to Amazon Linux 2 for ECS and thus use the newest teleport modules (with systemd)

File: monitoring/main.tf [hunk 2]
```diff
@@ -28,106 +31,111 @@ EOF
   aws_statistics: [Minimum, Maximum, Average, Sum]
 EOF
 
-  elasticsearch_monitring_template = "[${data.template_file.elasticsearch_exporter.rendered}${data.template_file.monitoring.rendered}]"
 
-  monitring_template = "[${data.template_file.monitoring.rendered}]"
+elasticsearch_monitring_template = "[${data.template_file.elasticsearch_exporter.rendered}${data.template_file.monitoring.rendered}]"
```
Comment (duboisph) [inline monitoring/main.tf]: While at it, can you fix these typos?

File: monitoring/main.tf [hunk 3]
```diff
@@ -28,106 +31,111 @@ EOF
...
 
-  containers_links = <<EOF
+monitring_template = "[${data.template_file.monitoring.rendered}]"
```
Comment (duboisph) [inline monitoring/main.tf]: Same here, typo.

**Review by duboisph — APPROVED** [review]
